### ISSUE TITLE
Basho closed

### ISSUE THREAD

**Issue by matthewvon** [basho/leveldb#229]
A small note that Basho has closed down.  I periodically review issues and will try to help.  Feel free to ping me directly via github's messaging.  

Life is busy for me, but I do plan to finish the shadow tables code and post in a cloned repository.  Likely be at least September before that happens.  At that time, I will restore hot backup code to the open source base.  (again, contact me directly if you have urgent need for either)

Matthew

**Comment by randysecrist** [comment]
A pleasure working with you sir.

**Comment by lemenkov** [comment]
Too sad to hear that. Anyway let's hope Riak survives all that mess.

**Comment by dyu** [comment]
> "At that time, I will restore hot backup code to the open source base

So I guess it was intentionally removed from the open source version?
Anyway, hot backup has always been available in [hyperleveldb](https://github.com/rescrv/HyperLevelDB)

**Comment by haneefmubarak** [comment]
@matthewvon could you tag the existing code as a release version? Thanks!

**Comment by matthewvon** [comment]
@haneefmubarak Nope.  No write access.  I am only a guest in this repository now.  My apologies.  "devel" is shippable.

**Comment by haneefmubarak** [comment]
It's all good, thanks anyways @matthewvon.

@russelldb wanna do like everyone here a solid and do the same (tag the current existing code as a release version)? Thanks!

**Comment by russelldb** [comment]
@haneefmubarak can you explain why?

**Comment by haneefmubarak** [comment]
@russelldb: so as @matthewvon mentioned, the code on branch `develop` is shippable as is.

However, when using this as a dependency of a project, it is useful to be able to refer to a released version (instead of just saying grab commit `xxxyyyzzz` or worse yet `HEAD`). Seeing as there have been significant (important and useful) changes since the last release and it does not appear that there will be more of such changes in the immediate future, making a release is the right thing to do.

**Comment by russelldb** [comment]
@haneefmubarak will talk to bet365 people (who own the repo) on Monday and figure something out. Can't see a reason why we can't tag it for you, though.

**Comment by russelldb** [comment]
@haneefmubarak we are about to make a release of riak though, and another coming in later summer/autumn (probably)

**Comment by haneefmubarak** [comment]
@russelldb awesome!
